analyzers/tests/SonarAnalyzer.UnitTest/Rules/Utilities/SymbolReferenceAnalyzerTest.cs

```diff
@@ -187,6 +188,30 @@ public void Verify_UnchangedFiles(string unchangedFileName, bool expectedProtobu
             }
         }
 
+#if NET
+
+        [TestMethod]
+        public void Verify_Razor() =>
+            // Currently only the symbols from .cs files are computed since .razor support is not yet implemented.
+            CreateBuilder(ProjectType.Product, "Razor.razor", "ToDo.cs")
```

Comment:
Educational: Why you decided to extend the CreateBuilder method with params instead of doing:
```cs
CreateBuilder(ProjectType.Product)
    .AddPaths("Razor.razor", "ToDo.cs")
    .WithConcurrentAnalysis(false)
```

Reply:
I think the code is easier to read this way. The logic of `CreateBuilder` already uses a file name to detect the language. Moving it to the `.AddPaths` would have meant completely refactoring the existing code. I could have called `.AddPath` to add the second file but then the files would have been set in two different ways, and that would have made the code more complicated.